libcaf_test/caf/test/reporter.hpp

```diff
@@ -76,6 +76,9 @@ class CAF_TEST_EXPORT reporter {
 
   virtual void verbosity(unsigned level) = 0;
 
+  /// Disables the colored output mode for the reporter
+  virtual void disable_colored(bool plain_output) = 0;
```

Comment:
"colored" is an adjective (or past tense). It can't stand on its own like this. I'd suggest `disable_colors` or `disable_colored_output` instead.

When naming the function in this active form, I find it illogical that it takes a parameter at all. So either stick to the `disable_*` command verb and take no argument or rename it to something like `colored_output(bool value)`.